content/guides/hosting/hosting-options/self-managed/kubernetes-operator/_index.md

```diff
@@ -465,28 +465,67 @@ global:
     secretKey: HDKYe4Q...JAp1YyjysnX
 ```
 
+The `kmsKey` must be `null`.
```

Comment:
I'm confused -- if it must be null, then why have the key at all? In the example for AWS above, the `kmsKey` is the empty string. Must it be empty for S3 and null for S3-compatible storage that is not on AWS?